static/manual_body.Rmd

```diff
@@ -1,237 +0,0 @@
----
```

Comment:
This script generates the manual_body.html file, so I don't think you can delete it.  (this is where all the recent changes in manual were carried out).